test/rest.browser.test.js

```diff
@@ -300,6 +300,185 @@ describe('strong-remoting-rest', function(){
     });
   });
 
+    describe('Checks argument matches enum', function () {
+      describe('in top level', function () {
+        it(', should not allow parameters that are required and not found in enum', function(done) {
```

Comment:
@tfemiani - Do you mean our test code, because this is in StrongLoop, so we should match their conventions